release/src/version-helpers.ts

```diff
@@ -232,32 +232,33 @@ export function versionSort(a: string, b: string) {
   return 0;
 }
 
-export function getLastReleaseFromTags(tags: Tag[]) {
+export function getLastReleaseFromTags({tags, ignorePatches = false}: { tags: Tag[], ignorePatches?: boolean }) {
   return tags
     .map(tag => tag.ref.replace('refs/tags/', ''))
     .filter(tag => !isRCVersion(tag)) // we want to ignore RC tags because release notes should be cumulative
-    .filter(v => !isPatchVersion(v)) // ignore patch versions since we don't release notes for them
+    .filter(ignorePatches ? v => !isPatchVersion(v) :  () => true) // ignore patch versions since we don't release notes for them
```

Comment:
We should probably move this comment to where we pass the argument rather, in `checkMilestoneForRelease`. This line doesn't really need explanation.